pkg/network/ebpf/c/protocols/classification/protocol-classification.h

```diff
@@ -64,6 +65,25 @@ static __always_inline bool is_protocol_classification_supported() {
     return val > 0;
 }
 
+// updates the the protocol stack and adds the current layer to the routing skip list
+static __always_inline void update_protocol_information(usm_context_t *usm_ctx, protocol_stack_t *stack, protocol_t proto) {
+    set_protocol(stack, proto);
+    usm_ctx->routing_skip_layers |= proto;
+}
+
+static __always_inline void classify_api_protocols(usm_context_t *usm_ctx, protocol_stack_t *protocol_stack, struct __sk_buff *skb, skb_info_t *skb_info) {
+    grpc_status_t status = is_grpc(skb, skb_info);
+    switch (status) {
+    case GRPC_STATUS_GRPC:
+        update_protocol_information(usm_ctx, protocol_stack, PROTOCOL_GRPC);
+        // If we found GRPC, we fallthrough to mark the stack as fully classified
+    case GRPC_STATUS_NOT_GRPC:
+        mark_as_fully_classified(protocol_stack);
```

Comment:
add an explicit comment for the reason you fall-through to  `GRPC_STATUS_UNKNOWN` case.
while it might seems reasonable to you, each code addition in the following case might cause troubles to the next person to handle the code.

Reply:
We probably can just remove the `UNKNOWN` case actually, since it effectively does nothing.